include/lsst/meas/extensions/photometryKron.h

```diff
@@ -36,7 +36,7 @@
 
 namespace lsst { namespace meas { namespace extensions { namespace photometryKron {
 
-struct KronAperture;
+class KronAperture;
```

Comment:
clang complains if a class is forward declared as a `struct` and then declared and implemented  as a `class`, or visa versa.